homeassistant/components/eight_sleep/sensor.py

```diff
@@ -263,14 +263,26 @@ def device_state_attributes(self):
             bed_temp = None
 
         if "current" in self._sensor_root:
-            state_attr[ATTR_RESP_RATE] = round(self._attr["resp_rate"], 2)
-            state_attr[ATTR_HEART_RATE] = round(self._attr["heart_rate"], 2)
+            try:
+                state_attr[ATTR_RESP_RATE] = round(self._attr["resp_rate"], 2)
+            except TypeError:
+                state_attr[ATTR_RESP_RATE] = None
```

Comment:
Our standard is to not add the item at all to the state attributes if the value is `None` or not available.

Reply:
How would that work from the end user perspective if the value is intermittent?  The underlying problem is sometimes the API the data is sourced from doesn't populate the value on a given call, but it's possible on the next interval it will populate correctly and show up.

Reply:
Templates need to check of the item is present.